pkg/resources/configmaps.go

```diff
@@ -229,6 +237,8 @@ func BuildConfigMap(instance *operatorv1alpha1.AuditLogging, name string) (*core
 			break
 		}
 		dataMap[QRadarConfigKey] = dq.Value
+	case FluentdDaemonSetName + "-" + HTTPIngestConfigName:
+		dataMap[HTTPIngestURLKey] = "https://common-audit-logging" + "." + InstanceNamespace + ":9880/icp-audit.http"
```

Comment:
Use `defaultHTTPPort` for the port. Not required, but maybe we could create a var to hold `icp-audit.http` in case we change it in the future?

Reply:
ok